guides/release-notes.md

```diff
@@ -7,6 +7,14 @@ We love hearing your feedback! You can help us improve the docs by reporting any
 ![Was this helpful, yes/no buttons](../media/feedback.png)
 
 ---
+### Deprecation Notice: Workflows API
```

Comment:
It's not really a deprecation notice. Maybe "Removed: Workflows v2 API" (specifying v2 since the crm-backend API will continue to work for the time being)

Reply:
Maybe we can say that it was experimental/developer preview?
And I agree, need to specify V2.